testsuite/features/step_definitions/lock_packages_on_client.rb

```diff
@@ -85,19 +86,16 @@
                 "span[@class='label label-info' and contains(text(), 'Unlocking...')]]"
   matches = all(:xpath, xpath_query)
 
-  raise if matches.size != pkgs.size
+  raise "Matches count #{matches.size} is different than packages count #{pkgs.size}" if matches.size != pkgs.size
```

Comment:
this is US English, a version that works in UK English too is "different from".

(okay, this one is _realy_ a nitpick :smile: )